build/l10n/xfail.list

```diff
@@ -114,16 +114,15 @@ apps/settings/js/utils.js
 apps/settings/test/unit/firefox_accounts_panel_test.js
 apps/sharedtest/test/unit/l10n/l10n_test.js
 apps/sharedtest/test/unit/lazy_l10n_test.js
-apps/sms/js/activity_handler.js
```

Comment:
Looks like we still use ```mozL10n.get``` in activity_handler https://github.com/mozilla-b2g/gaia/blob/e71b0cb1c8b7e94e090e0e306b6b56f9b6bb625e/apps/sms/views/shared/js/activity_handler.js#L313 ?